galleon-pack/galleon-shared/src/main/resources/license/licenses.xml

```diff
@@ -558,16 +558,6 @@
                 </license>
             </licenses>
         </dependency>
-        <dependency>
-            <groupId>io.opentelemetry</groupId>
-            <artifactId>opentelemetry-api-logs</artifactId>
-            <licenses>
-                <license>
-                    <name>Apache License 2.0</name>
-                    <url>http://www.apache.org/licenses/LICENSE-2.0</url>
-                </license>
-            </licenses>
-        </dependency>
```

Comment:
I expect there may need to be more stuff done to this file, although maybe not as how this works is being redone.